website/docs/models/index.mdx

```diff
@@ -108,12 +108,12 @@ In the `sm`/`md`/`lg` models:
 
 #### CNN/CPU pipelines with floret vectors
 
-The Finnish, Korean and Swedish `md` and `lg` pipelines use
-[floret vectors](/usage/v3-2#vectors) instead of default vectors. If you're
-running a trained pipeline on texts and working with [`Doc`](/api/doc) objects,
-you shouldn't notice any difference with floret vectors. With floret vectors no
-tokens are out-of-vocabulary, so [`Token.is_oov`](/api/token#attributes) will
-return `False` for all tokens.
+The Croatian, Finnish, Korean, Slovenian, Swedish and Ukrainian `md` and `lg`
+pipelines use [floret vectors](/usage/v3-2#vectors) instead of default vectors.
+If you're running a trained pipeline on texts and working with [`Doc`](/api/doc)
+objects, you shouldn't notice any difference with floret vectors. With floret
+vectors no tokens are out-of-vocabulary, so
+[`Token.is_oov`](/api/token#attributes) will return `False` for all tokens.
```

Comment:
Should we link to the floret vectors blog post too?

Reply:
Hmm, I wonder if there's a better usage-focused place to link to, but I'm not sure if there is.

Reply:
https://github.com/explosion/floret? Not that that's very usage-focused either.

Reply:
I think the v3-2 section is good enough for floret usage (?), I meant more spacy usage-focused.

Reply:
I think it's fine, yes.